src/cmd/flux-kvs.c

```diff
@@ -56,6 +56,16 @@ static void dump_kvs_dir (const flux_kvsdir_t *dir, bool Ropt, bool dopt);
 
```

Comment:
typo in commit comment, "strong" -> "string"

Reply:
hmm, comment appears to be attached to file and not a commit.  This is in commit https://github.com/flux-framework/flux-core/pull/1257/commits/f93f7768ffc5aef5901118bd48541c826d0c7966